### DISCUSSION TITLE
How to identify prior knowledge in newly labeled dataset

### DISCUSSION THREAD

**Discussion by HvAlem** [asreview/asreview#1932] [Q&A] [answered]
Hi!

My team and I have been working on a review, and we have recently been asked to update the search. We used the labels from the first search (20 relevant, 12777 irrelevant) as prior knowledge. Because the second search was screened with a new screener than the first one, i want to recalculate the Cohen's Kappa. The problem is, that I cannot seem to get rid of the prior knowlegde, as the new data now has the same (ir)relevant labels. Is there a way to get a 'clean' file without the prior knowlegde? 

Thank you in advance!

Hannah

**Comment by Rensvandeschoot** [comment]
Are you using Excel/CSV or RIS as export file? In both versions, there should be an identifier for prior knowledge. In the Excel file this is a column and in that RIS file it is part of the N1 field. See for more information: https://asreview.readthedocs.io/en/latest/progress.html#export-results. With this information you should be able to de-select the records used for prior knowledge, right?

**Comment by HvAlem** [comment, marked as answer]
We used excel. Thank you so much! We finally managed to remove the prior knowledge – you made our Monday! Typical that the solution turned out to be so easy 😊

Met vriendelijke groet,

Hannah van Alem
Promovendus
